### PR TITLE
added tokeet detector

### PR DESCRIPTION
**Pull request by lonmarsDev**

### CODE REVIEW

File: pkg/detectors/tokeet/tokeet.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeet"}) + `\b([0-9Aa-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`)
```
Comment (roxanne-tampus) [inline pkg/detectors/tokeet/tokeet.go:24]: suggestion: prefer removal "A" here for a bit stricter validation `...\b([0-9Aa-f]{8}...`

File: pkg/detectors/tokeet/tokeet.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	idPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeetId"}) + `\b([0-9.]{15})\b`)
```
Comment (roxanne-tampus) [inline pkg/detectors/tokeet/tokeet.go:25]: `tokeetId` is unlikely to be found, recommended to just use `tokeet`.

File: pkg/detectors/tokeet/tokeet.go [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+				DetectorType: detectorspb.DetectorType_Tokeet,
+				Raw:          []byte(resMatch),
+			}
+			if verify {
+				req, err := http.NewRequestWithContext(ctx, "GET", "https://capi.tokeet.com/v1/user?account="+resIdMatch, nil)
+				if err != nil {
+					continue
+				}
+				req.Header.Add("Accept", "application/json")
+				req.Header.Add("Authorization", fmt.Sprintf("%s", resMatch))
```
Comment (roxanne-tampus) [inline pkg/detectors/tokeet/tokeet.go:64]: nit: Prefer to remove fmt.Sprintf here: `fmt.Sprintf("%s", resMatch`) and we can just use `..("Authorization", resMatch)...`

File: pkg/detectors/tokeet/tokeet.go [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+				continue
+			}
+			resIdMatch := strings.TrimSpace(idmatch[1])
+
+			s1 := detectors.Result{
+				DetectorType: detectorspb.DetectorType_Tokeet,
+				Raw:          []byte(resMatch),
+			}
+			if verify {
+				req, err := http.NewRequestWithContext(ctx, "GET", "https://capi.tokeet.com/v1/user?account="+resIdMatch, nil)
```
Comment (roxanne-tampus) [inline pkg/detectors/tokeet/tokeet.go:59]: nit: Prefer fmt.Sprintf here: `fmt.Sprintf("https://capi.tokeet.com/v1/user?account=%s", resIdMatch)`

File: pkg/detectors/tokeet/tokeet.go [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeet"}) + `\b([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`)
+	idPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeet"}) + `\b([0-9.]{15})\b`)
```
Comment (roxanne-tampus) [inline pkg/detectors/tokeet/tokeet.go:25]: Can you please enhance this regex as this can be a bit stricter by using `.` as separator?

**Review by dustin-decker — APPROVED** [review]

**Pull request merged**